src/components/PageNotFound.js

```diff
@@ -12,16 +18,27 @@ class PageNotFound extends Component {
   render() {
     return (
       <div className="page-not-found">
-        <div>
-          <div className="page-not-found__title">Error 404</div>
-          <div className="page-not-found__content">Woops. Looks like this page doesn't exist </div>
+        <div className="page-not-found__container">
+          <img
+            className="page-not-found__image"
+            src={COW_PIC_SRC}
+            alt="cow-img"
+          />
+          <div className="page-not-found__title">Page Not Found</div>
+          <div className="page-not-found__content">
+            This is not the web page you are looking for. <br />
+            You will be redirected to the My Range Application home page within 10 seconds.
```

Comment:
this would be better semantically as  `<div className="page-not-found__content"><p>This is not...</p><p>You will be...</p></div>`